added to-dos (#6)
johntremblay/ift-6756-alphazero: to_dos.txt

# TODO AS OF MARCH 28:

> - Benchmark comparison between models: Win rate over random players (100 games) after 3 update of the new model + 1000 game after 15 update

Reviewer: ```suggestion
- Benchmark comparison between models: Win rate over random players (100 games) after 3 update of the new model + 1000 game after 15 update (JT)
```

> - NN network training progression (plotting, loss metric, etc.).

Reviewer: ```suggestion
- NN network training progression (plotting, loss metric, etc.). (JT)
```

> - Saving of model configurations (NN and alpha zero), GPU, game played, time of training, etc.

Reviewer: ```suggestion
- Saving of model configurations (NN and alpha zero), GPU, game played, time of training, etc. (JT)
```

# TODO AS OF MARCH 28:

> - Benchmark comparaison between models: Win rate over random players (1000 games)? Do we want to evaluate this only at

Reviewer: ```suggestion
- Benchmark comparison between models: Win rate over random players (100 games) after 3 update of the new model + 1000 game after 15 update
```

> the end of training or 2-3 iterations during the process to see if one might learn faster than another?

Reviewer: ```suggestion
```
